From 850c7c28815053bd52ef0305ef0d161d91b71685 Mon Sep 17 00:00:00 2001
From: Rhys-T <108157737+Rhys-T@users.noreply.github.com>
Date: Tue, 27 Jan 2026 12:18:18 -0500
Subject: [PATCH] python3Packages.pysdl2: skip broken test

SDL3 >= 3.3.4 now returns 'unsupported' from SDL_SetWindowMouseRect:
https://github.com/libsdl-org/SDL/commit/2a873be9cdb35972dc8bacd12653aafc02e835de

This recently had to be addressed for pygame-* as well, in #483604.

Resolves #483673.
---
 pkgs/development/python-modules/pysdl2/default.nix | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pkgs/development/python-modules/pysdl2/default.nix b/pkgs/development/python-modules/pysdl2/default.nix
index dfcc76bb132b9..66fb3a4664aa4 100644
--- a/pkgs/development/python-modules/pysdl2/default.nix
+++ b/pkgs/development/python-modules/pysdl2/default.nix
@@ -85,6 +85,9 @@ buildPythonPackage rec {
     # AssertionError:
     # clip: Could not set clip rect SDL_Rect(x=2, y=2, w=0, h=0)
     "test_SDL_GetSetClipRect"
+
+    # AssertionError: That operation is not supported
+    "test_SDL_GetSetWindowMouseRect"
   ];
 
   meta = {
